bundles/org.openhab.binding.knx/README.md

```diff
@@ -235,18 +235,23 @@ For UoM support see the explanations of the `number` channel.
 #### Group Address Notation
 
 ```text
-<config>="[<dpt>:][<]<mainGA>[[+[<]<listeningGA>][+[<]<listeningGA>..]]"
+<config>="[<dpt>:][<>]<mainGA>[[+[<]<listeningGA>][+[<]<listeningGA>..]]"
 ```
 
 where parts in brackets `[]` denote optional information.
 
+**Each configuration parameter has a `mainGA` where commands are written to and optionally several `listeningGA`s.**
+
+`mainGA` also listens to incoming packets, unless prefixed with a `>` character.
+This is recommended if you have dedicated status group addresses which are added as `listeningGA`s.
```

Comment:
```suggestion
This is recommended if you have a dedicated status group address which is added as `listeningGA`.
```

I can't imagine a case where a single channel config has more than a single status GA. Do you?